cronet/README.md

```diff
@@ -17,13 +16,26 @@ Some advantages of using Cronet with gRPC:
 * Robust to Android network connectivity changes
 * Support for [QUIC](https://www.chromium.org/quic)
 
-Cronet jars are available on Google's Maven repository. See the example app at
-https://github.com/GoogleChrome/cronet-sample/blob/master/README.md. To use
-Cronet with gRPC, you will need to copy the gRPC source files contained in this
-directory into your application's code, as we do not currently provide a
-`grpc-cronet` dependency.
+Since gRPC's 1.24 release, the `grpc-cronet` package provides access to the 
+`CronetChannelBuilder` class. Cronet jars are available on Google's Maven repository. 
+See the example app at https://github.com/GoogleChrome/cronet-sample/blob/master/README.md.
 
-To use Cronet, you must have the `ACCESS_NETWORK_STATE` permission set in
+## Example usage:
+
+In your app module's `build.gradle` file, include a dependency on both `grpc-cronet` and the 
+Google Play Services Client Library for Cronet
+
+```
+implementation 'io.grpc:grpc-cronet:1.24.0'
```

Comment:
How do we maintain the version in the doc for new releases? Add 
```
$ ${EDITOR:-nano -w} cronet/README.md
```

to RELEASING.md?

Reply:
Seems we do not sync the version number in README.md. 

I saw grpc-android's doc is also outdated. We may want to add that to RELEASING.md.

Reply:
Which file is grpc-android's doc you mentioned?

Reply:
https://github.com/grpc/grpc-java/blob/master/documentation/android-channel-builder.md#example-usage

Reply:
That is up to date. You only need to bump its version during releasing process.

Reply:
Oh, you are right. That's done in tagging the release. We could do the same for cronet by adding 
```
$ ${EDITOR:-nano -w} cronet/README.md
```

Reply:
Added the version bump command to RELEASING.md.